### ISSUE TITLE
Do we still need the infinity behavior of angleToleranceInterval?

### ISSUE THREAD

**Issue by BLFiedler** [phetsims/quadrilateral#214]
I'm curious as to whether our shape detection changes have eliminated the need for angleToleranceInterval to adjust to infinity/-infinity when dragging the sides. Becoming a parallelogram now has more stringent requirements (side length and angle comparison).

Could we create a one-off version that eliminates this behavior just to test?

**Comment by jessegreenberg** [comment]
Sure! In this version I removed the "infinity" behavior. The tolerance still gets bigger with multitouch and tangible connection.

I don't understand how it could be removed, parallel state is a dependency of the shape detection. That being said I wasn't able to break it in ~5 minutes of play. Over to you @BLFiedler 

https://phet-dev.colorado.edu/html/quadrilateral/1.0.0-dev.63/phet/quadrilateral_en_phet.html

**Comment by BLFiedler** [comment]
Thanks! I likewise cannot break it yet.

I believe the infinity behavior was needed to mask some of the small differences in angles changes that I described in #210 and #212. I'm having a hard time articulating why it was so specifically a problem for the parallelogram <-> trapezoid transition, but it is related to the reliance on a single shared requirement. I think I indicated in some issue..., but otherwise when we were working out another problem in a meeting that when dragging a parallelogram or trapezoid to extreme angles, the angle requirements to be parallel were going in and out of tolerance, but the side length and individual angle requirements were not being met. We now have more stringency for both of those as instantiated in #188. 

Let's sanity check the next time we meet. Afterward, we'll see about merging the changes.

**Comment by BLFiedler** [comment]
Let's go ahead and push this to master. No more infinity behavior for angleToleranceInterval with the new shape detection.

**Comment by jessegreenberg** [comment]
Just noting my understanding of why we can remove this now:

The "infinity" behavior was in place to make sure that when two sides *start* in parallel, dragging one of the sides cannot make those sides go *out* of parallel (and vice versa). Geometrically, that should not be possible but it can happen because of tolerance intervals.

Removing the *infinity* behavior, we are allowing this to happen! But that is OK. Maintaining parallel-ness is no longer as critical for this sim as long as that state change does not impact the detected shape name. And after https://github.com/phetsims/quadrilateral/issues/188, it does not.

**Comment by jessegreenberg** [comment]
OK! This implementation (and references to this behavior throughout sim documentation) have been removed.

@BLFiedler anything else here? In meeting we discussed that we wanted to keep the dynamic `angleToleranceInterval` behavior for multitouch cases. I can't remember why that was decided, can you comment @BLFiedler?

**Comment by BLFiedler** [comment]
I believe after extensive testing for other issues and not running into this issue at the boundary or a parallelogram/trapezoid, that we are alright to close this.

We'll open another issue for dynamic tolerances as pertaining to multitouch, computer vision, and external devices.
